.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           - rust_version: nightly
             run_tests: true
           # Minimal supported rustc version
-          - rust_version: 1.46.0
+          - rust_version: 1.47.0
```

Comment:
why change to MSRV?

Reply:
Deriving `Debug` for fixed-sized arrays needs 1.47.

Reply:
Specifically the big `[u8; 448]` array.